src/transport/raw/WiFiPAF.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2024 Project CHIP Authors
+ *    Copyright (c) 2025 Project CHIP Authors
```

Comment:
Again, why is this nuking the old copyright date?

Reply:
I will change it